package.json

```diff
@@ -85,7 +87,7 @@
   "scripts": {
     "postpack": "rm -f oclif.manifest.json",
     "pretest": "tslint -p . -t stylish && tslint -p test -t stylish && prettier -c \"**/*.ts\"",
-    "prettier": "prettier --write \"**/*.ts\"",
+    "prettier": "tslint -p . -t stylish --fix && tslint -p test -t stylish --fix && prettier --write \"**/*.ts\"",
```

Comment:
Please rename the script to reflect what it does, e.g. `fix:formatting` or `lint-fix`

Reply:
good point